lib/tire/index.rb

```diff
@@ -335,7 +335,11 @@ def remove(*args)
       end
       raise ArgumentError, "Please pass a document ID" unless id
 
-      url    = "#{self.url}/#{type}/#{Utils.escape(id)}"
+      options ||= {}
+      options.select! { |k, | [:routing, :parent, :replication, :consistency, :refresh, :timeout].include? k }
+      params_encoded = options.empty? ? '' : "?#{options.to_param}"
+
+      url    = "#{self.url}/#{type}/#{Utils.escape(id)}#{params_encoded}"
```

Comment:
`Utils.escape(type)`

Reply:
This is how every method in tire is currently written.

Reply:
Yes and, I found where it was escaped though:
https://github.com/outstand/tire/blob/18942917a07ed4ea278f7b3686f30dc187608d1b/lib/tire/index.rb#L329

I'm assuming that getting type from document is already escaped?
https://github.com/outstand/tire/blob/18942917a07ed4ea278f7b3686f30dc187608d1b/lib/tire/index.rb#L333

This is the line that made me think about it:
https://github.com/outstand/tire/blob/18942917a07ed4ea278f7b3686f30dc187608d1b/lib/tire/index.rb#L354

Reply:
Fixed.

Reply:
Unfixed!